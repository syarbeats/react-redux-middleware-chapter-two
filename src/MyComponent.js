import React, { Component } from 'react';

class MyComponent extends Component {
    render() {
        return (
            <div className="card">
                <div className="card-body">
                    <h4 className="card-title">Right Side</h4>
                    <p className="card-text">Some example text. Some example text.</p>
                    <a href="#" className="card-link">Card link</a>
                    <a href="#" className="card-link">Another link</a>
                </div>
            </div>
        );
    }
}
export default MyComponent;
