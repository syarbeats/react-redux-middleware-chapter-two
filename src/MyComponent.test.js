import React from 'react';
import MyComponent from './MyComponent';
import renderer from 'react-test-renderer';

describe('MyComponent component', () => {
    it('matches the snapshot', () => {
        const tree = renderer.create(<MyComponent/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})