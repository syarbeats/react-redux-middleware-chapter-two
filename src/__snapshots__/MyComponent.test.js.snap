// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent component matches the snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <h4
      className="card-title"
    >
      Right Side
    </h4>
    <p
      className="card-text"
    >
      Some example text. Some example text.
    </p>
    <a
      className="card-link"
      href="#"
    >
      Card link
    </a>
    <a
      className="card-link"
      href="#"
    >
      Another link
    </a>
  </div>
</div>
`;
